independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/UniInvoker.java

```diff
@@ -25,6 +27,14 @@ public <R> Uni<R> method(String name, Entity<?> entity, GenericType<R> responseT
             public CompletionStage<R> get() {
                 return invoker.method(name, entity, responseType);
             }
+        }).onFailure().transform(new Function<>() {
+            @Override
+            public Throwable apply(Throwable t) {
+                if (t instanceof ProcessingException) {
+                    return t.getCause();
```

Comment:
Might be better to also check that the cause is not null.

Reply:
Good point